### DISCUSSION TITLE
Can we have a source distribution in pypi

### DISCUSSION THREAD

**Discussion by Stonre** [pythonspeed/filprofiler#329] [Ideas]
Hi, I am wondering whether we can have a source distribution for filprofiler which can make it much easier for everyone to import it to their gitfarm for usage? Thanks.

**Comment by itamarst** [comment]
You can actually install from the source releases on GitHub, probably. E.g. `pip install https://github.com/pythonspeed/filprofiler/archive/refs/tags/2022.01.1.tar.gz` might work. I'll also open an issue for upload sdists to PyPI.

**Reply by Stonre** [reply]
Thanks itamarst! Looking forward to source distribution in Pypi!
